clang/lib/CodeGen/CGOpenMPRuntimeNVPTX.cpp

```diff
@@ -33,6 +33,13 @@ CGOpenMPRuntimeNVPTX::CGOpenMPRuntimeNVPTX(CodeGenModule &CGM)
 }
 
 llvm::Value *CGOpenMPRuntimeNVPTX::getGPUWarpSize(CodeGenFunction &CGF) {
+  if (CGF.getTarget().getTriple().isAMDGCN()) {
```

Comment:
This is probably a missing patch in trunk - getGPUWarpSize should resolve to nvptx or amdgcn in the same fashion as the other factored out functions